client/conf/server.properties.in

```diff
@@ -38,7 +38,11 @@ https.port=8443
 
 # The keystore and manager passwords are assumed to be same.
 https.keystore=/etc/cloudstack/management/cloud.jks
+# If you want to encrypt the password follow the steps mentioned at:
+# http://docs.cloudstack.apache.org/projects/archived-cloudstack-administration/en/latest/management.html?highlight=jasypt#changing-the-database-password
 https.keystore.password=vmops.com
+# If an encrypted password is used, specify the encryption type - valid types: file, env (set environment variable CLOUD_SECRET_KEY), web
+# db.cloud.encryption.type=none
```

Comment:
"_db.cloud.encryption.type_" parameter name doesn't suit for the keystore password encryption. May be, can use "_https.keystore.encryption.typ_e" (or "_password.encryption.type_").